scripts/xsslint/xsslint/linters.py

```diff
@@ -448,7 +448,8 @@ def _check_javascript_interpolate(self, file_contents, results):
 
         """
         # Ignores calls starting with "StringUtils.", because those are safe
-        regex = re.compile(r"(?<!StringUtils).interpolate\(")
+        # XXX: this will false positive on any function that ends with 'interpolate'...
```

Comment:
Leftover comment?

Reply:
Not really leftover, since it's still relevant but out of scope. I can remove it if you like.

Reply:
Removed, and also made the xsslint `escape` check more accurate (will also check for `window.escape(`).

Reply:
Sorry, I don't mind it being there. I was just confused what "XXX" meant...?

Reply:
I've always used that as a variant to TODO, FIXME, et al, usually to mean that it's potentially buggy. https://stackoverflow.com/a/1452945/